data/hai ships.txt

```diff
@@ -262,7 +262,7 @@ ship "Shield Beetle"
 	explode "large explosion" 50
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
-	description "The Shield Beetle is a versatile warship used by the Hai to guard their merchant convoys."
+	description "The Shield Beetle is the peaceful Hai's only heavy warship. It is a versatile warship used by the Hai to guard their merchant convoys, and serves as a their main defender against Unfettered aggression from the North."
```

Comment:
`The Shield Beetle is the peaceful Hai's only heavy warship.` -> `For many millenia now, the Shield Beetle has been the only heavy warship at the disposal of the Hai.`
"peaceful Hai" sounds really corny to have in game.